tests/integration/tests/direction.js

```diff
@@ -80,14 +82,13 @@ test('route flag', async () => {
   expect(directionEndInput).toEqual('');
 });
 
-
 test('destination', async () => {
-  expect.assertions(3);
-  await page.goto(`${APP_URL}/${ROUTES_PATH}/?destination=latlon:47.4:7.5@Monoprix Nice`);
+  responseHandler.addPreparedResponse(mockPoi1, new RegExp(`places/${mockPoi1.id}`));
 
+  expect.assertions(2);
+  await page.goto(`${APP_URL}/${ROUTES_PATH}/?destination=${mockPoi1.id}`);
   await page.waitForSelector('#itinerary_input_origin');
-  const smallToolBar = await page.waitForSelector('.top_bar--small');
```

Comment:
In the three cases I removed the testing of this stuff for simplification, as it's not related to the input fields and is already tested in another case.